rholang/reference_doc/sends_and_receives/README.md

```diff
@@ -1,17 +1,62 @@
 ## Sends and Receives
 
-### Tuple space
-- Concrete, technical explanation of the tuple space
+### Tuplespace (RSpace)
+Rholang is based on the rhoVM whose computational model is centered around processes sending messages to one another to synchronize when necessary. These operations are coordinated through the tuplespace.
+Although it is unique from the Turing or Von Neuman architectures that many programmers will be familiar with, it is quite understandable. All tuplespace operations can be built up from three building blocks.
+
+![building blocks](buildingBlocks.png)
+
+* A channel on which messages are send and received
+* Data to be sent (includes [primitive types](../primatives/README.md), [data structures](../data_structures/README.md), or even entire processes. )
+* Continuations which spawn new processes when a message is received
 
 ### Sends
-- Content that can be sent
-- Difference between one-tuple and single process in the syntax
-- Persistent sends
-- Send a process (receive a name)
+![Sends](send.png)
+Sending a name associates the sent data with the channel in the tuplespace. It will wait there forever until it matches with a receive on the same channel.
```

Comment:
Technically you send a process, but in the tuple space it sits there as a name. If you send a tuple of processes, it sits there as a tuple of names.